content/ngf/installation/installing-ngf/manifests.md

```diff
@@ -11,9 +11,6 @@ docs: DOCS-1429
 
 Learn how to install, upgrade, and uninstall NGINX Gateway Fabric using Kubernetes manifests.
 
-{{< important >}} NGINX Plus users that are upgrading from version 1.4.0 to 1.5.x need to install an NGINX Plus JWT
-Secret before upgrading. Follow the steps in the [Before you begin](#before-you-begin) section to create the Secret, which is referenced in the updated deployment manifest for the newest version. {{< /important >}}
-
```

Comment:
This isn't obvious from the pull request diff, but this is a redundant warning: it exists at the top of the upgrade section.